src/api.tsx

```diff
@@ -81,4 +82,8 @@ export const createApiWrapper = <PeerMetadata, TrackMetadata>(
   setTargetTrackEncoding: (trackId, encoding) => {
```

Comment:
I know it is not your change but i suggest to add typing also here, if everywhere then everywhere :D